### DISCUSSION TITLE
Filecoin statistics on storage.filecoin.io not updated

### DISCUSSION THREAD

**Discussion by rajaspeet1997** [filecoin-project/community#607] [General]
Hi,

I follow the Filecoin network, and https://storage.filecoin.io/ is a great resource for this. Unfortunately, the stats do not update anymore. Does anyone have an alternative or knows perhaps if this is just temporary?

Kind regards

**Comment by ribasushi** [comment]
Whoops, this was not intended. I'll fix this in the next 24h or so.

Thank you for flagging!

**Reply by rajaspeet1997** [reply]
Thanks for you quick response. Just checked the website, but see it still shows data from 8 January 2023. Just for your information.

Hope u can fix it, cheers!
